packages/vsx-registry/src/browser/vsx-extensions-contribution.ts

```diff
@@ -173,6 +190,43 @@ export class VSXExtensionsContribution extends AbstractViewContribution<VSXExten
         }
     }
 
+    /**
+     * Given an extension, displays a quick pick of other compatible versions and installs the selected version.
+     *
+     * @param extension a VSX extension.
+     */
+    protected async installAnotherVersion(extension: VSXExtension): Promise<void> {
+        const extensionId = extension.id;
+        const currentVersion = extension.version;
+        const client = await this.clientProvider();
+        const extensions = await client.getAllVersions(extensionId);
+        const latestCompatible = await client.getLatestCompatibleExtensionVersion(extensionId);
+        let compatibleExtensions: VSXExtensionRaw[] = [];
+        if (latestCompatible) {
+            compatibleExtensions = extensions.slice(extensions.findIndex(ext => ext.version === latestCompatible.version));
+        }
+        const items: QuickPickItem[] = [];
+        compatibleExtensions.forEach(ext => {
+            let publishedDate = moment(ext.timestamp).fromNow();
+            if (currentVersion === ext.version) {
+                publishedDate += ' (Current)';
+            }
```

Comment:
For some reason I do not see `current` added as a description to the current extension version:

https://user-images.githubusercontent.com/40359487/140516403-4efc9849-cce1-4fca-b950-c315833b1841.mp4

I also verified when it is not the first entry in the list.